### PR TITLE
Guide for devs using scenario parameters

### PR DESCRIPTION
**Pull request by seb231**

### CODE REVIEW

File: doc/scenarios.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Alternative Scenarios
+
+This is a short primer for how to use some of the parameters available in the witan.send model in order to run _"alternative scenarios"_.
+
+Each of the paramater keys (e.g. `:splice-ncy`) can be found, and their values defined, in the [config.edn](https://github.com/MastodonC/witan.send/blob/master/data/demo/config.edn).
+
+More information on how to define a scenario can be found here (TBD).
```
Comment (seb231) [inline doc/scenarios.md]: This is currently [here](https://docs.google.com/document/d/1lQ2RrESpiyU5x2YUY8YvT287K4iq6NJw4jqHUIqC2VM/edit) (could a

File: doc/scenarios.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Each of the paramater keys (e.g. `:splice-ncy`) can be found, and their values defined, in the [config.edn](https://github.com/MastodonC/witan.send/blob/master/data/demo/config.edn).
+
+More information on how to define a scenario can be found here (TBD). 
+
+Here each scenario is simply defined in terms of the collection of parameters to use together, what arguments are expected and what the expected behaviuor is.
+
+### _“Ignore historic data before a specific calendar year for an age group”_
+
+* Parameters = `:filter-transitions-from` & `:splice-ncy`
+* Arguments = a calendar year and national curriculum year
```
Comment (mikeyford) [inline doc/scenarios.md]: It's a little unclear to me how :splice-ncy works. Does this filter the data above, equal to and above, below etc? Perhaps give example args for your NCY 11 example below?

File: doc/scenarios.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+* Defines a separate set of transition rates based on a defined start year and age
+* For example, historic transitions may data back to 2014, but a user may only want to use data for a specific age group from 2016
+Typically we see a significant change in the behaviour in those over NCY 11, as these are only recently included in SEND, so data prior to this change may not want to be included as it could skew the models results
+
+### _“Modify setting(s) transitions rates”_
+
+* Parameters = `:which-transitions?`, `:modify-transitions-by` and `:settings-to-change`
+* Arguments = a file defining the settings rates to be modified, a value modify the transitions by, and a vector containing one or multiple strings describing the possible type of transitions to be modified, e.g.
```
Comment (mikeyford) [inline doc/scenarios.md]: Do we have an example of the structure for this file to point to?

File: doc/scenarios.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  * “Joiners”
+  * “Leavers”
+  * “Movers-to”
+  * “Movers-from”
+* Defines single or multiple settings and type of transitions to modify the rates of, and by how much
+* For example, a user may want to modify how many individuals are joining to the setting “Mainstream” by halving the current rate.
```
Comment (mikeyford) [inline doc/scenarios.md]: What would this look like in the config?

File: doc/scenarios.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+# Alternative Scenarios
+
+This is a short primer for how to use some of the parameters available in the witan.send model in order to run _"alternative scenarios"_.
+
+Each of the paramater keys (e.g. `:splice-ncy`) can be found, and their values defined, in the [config.edn](https://github.com/MastodonC/witan.send/blob/master/data/demo/config.edn).
+
+More information on how to define a scenario can be found here (TBD).
```
Comment (otfrom) [inline doc/scenarios.md]: Can we create a card for the TBD bit so we don't forget it?

Comment (seb231) [reply doc/scenarios.md]: done https://www.pivotaltracker.com/story/show/159827443

**Pull request merged**